tests/test_ramp_fitting.py

```diff
@@ -1567,3 +1568,20 @@ def print_all_info(slopes, cube, optional):
     print_slopes(slopes)
     print_integ(cube)
     print_optional(optional)
+
+def test_compute_num_slices():
```

Comment:
This test should be moved to line 1358.  The test file is divided into different sections.  The bottom section is for debugging functions.  There is a test suite section that currently ends at 1358.